include/crpropa/magneticField/turbulentField/TurbulentField.h

```diff
@@ -0,0 +1,119 @@
+#ifndef CRPROPA_TURBULENTFIELD_H
+#define CRPROPA_TURBULENTFIELD_H
+
+#include "crpropa/magneticField/MagneticField.h"
+#include <cmath>
+
+namespace crpropa {
+/**
+ * \addtogroup MagneticFields
+ * @{
+ */
+
+/**
+ @class TurbulenceSpectrum
+ @brief Defines the energy spectrum of turbulence
+ */
+class TurbulenceSpectrum : public Referenced {
+  private:
+	const double Brms; /**< Brms value of the turbulent field (normalization) */
+	const double sIndex; /**< Spectral index for the inertial range, for example
```

Comment:
Shall we clarify at some point why we are sometime referring to a Kolmogorov spectrum with an index of 5/3 and sometime with 11/3? 

I know the difference come from the additional k^2 when you do the integral for an isotropic turbulence in spherical coordiantes. I am just not sure if we should include a comment somewhere as long as we have these two implementations in parallel.

Reply:
Right. I've added here the following:
```
/**< Spectral index for the inertial range, for example
                          s=5/3 for Kolmogorov spectrum; in some parts of the code this
                          parameter is referred by alpha which is the total 3D isotropic
                          spectrum with additional k^2 and the minus sign, e.g.,
                          for Kolmogorov: alpha = -(s + 2) */
```

Reply:
That's a good explanation.